README.md

```diff
@@ -545,6 +545,8 @@ make delete
 
 # 4. Running indexer or service locally
 
+**Ensure infrastructure for lambda(s) are deployed before trying to run locally. See Sections: [2](#2-getting-started), [3](#3-deployment)**
```

Comment:
```suggestion
While this method *does* run the service or indexer locally on your machine, it still requires that the cloud resources used by them are already deployed. See sections [2](#2-getting-started) and [3](#3-deployment) on how to do that.
```

and check if I botched the links. And wrap at 80.

Reply:
Done.